src/aap_eda/api/serializers/activation.py

```diff
@@ -620,12 +616,6 @@ class Meta:
             "skip_audit_events",
         ]
 
-    organization_id = serializers.IntegerField(
-        required=False,
-        allow_null=False,
-        validators=[validators.check_if_organization_exists],
-        error_messages={"null": "Organization is needed"},
-    )
```

Comment:
The newly added `OrganizationIdFieldMixin` mixin has `required=True` whereas this field (replaced by the mixin) has `required=False`; is the change correct here? If so, I'd at least mention that in the PR/commit message

Reply:
Hi @ptoscano the requirement is applied only for the creation, it does not require any field in PATCH unless it is explicitily set (because of `partial=True`)